project/src/hooks/useMap/useMap.ts

```diff
@@ -1,6 +1,7 @@
 import { MutableRefObject, useEffect, useState } from 'react';
-import {Map, TileLayer} from 'leaflet';
+import { Map, TileLayer } from 'leaflet';
 import { City } from '../../types/offer';
+import { MAP_TAILE_LAYER, MAP_TAILE_LAYER_ATTRIBUTIONS } from '../../const';
```

Comment:
Какая вероятность переиспользования этих констант? Я бы наверно держал все таки их "ближе к телу"
```suggestion
import { MAP_TILE_LAYER, MAP_TILE_LAYER_ATTRIBUTIONS } from '../../const';
```